src/datatypes.jl

```diff
@@ -367,19 +365,17 @@ mutable struct GDPData
     disjunctions::_MOIUC.CleverDict{DisjunctionIndex, ConstraintData{Disjunction}}
 
     # Indicator variable mappings
-    disjunction_to_indicators::Dict{DisjunctionIndex, Vector{LogicalVariableIndex}}
-    indicator_to_disjunction::Dict{LogicalVariableIndex, DisjunctionIndex}
     indicator_to_binary::Dict{LogicalVariableIndex, _MOI.VariableIndex}
     indicator_to_constraints::Dict{LogicalVariableIndex, Vector{DisjunctConstraintIndex}}
     constraint_to_indicator::Dict{DisjunctConstraintIndex, LogicalVariableIndex}
-    
-    # Map of disaggregated variables 
-    global_to_disjunct_variable::Dict{Tuple{_MOI.VariableIndex,_MOI.VariableIndex}, _MOI.VariableIndex}
-    global_to_disjunction_variables::Dict{Tuple{_MOI.VariableIndex,DisjunctionIndex}, Vector{_MOI.VariableIndex}}
 
     # Map of variable bounds
     variable_bounds::Dict{_MOI.VariableIndex, Tuple{Float64, Float64}} # TODO allow for other precision
 
+    # Reformulation variables and constraints
+    reformulation_variables::Set{_MOI.VariableIndex}
```

Comment:
Using a `Vector` will probably be more efficient.

Reply:
ok, changed